# 3-11: an intentional error?  If that's what you want, then let's make it happen.  T_T
myList = ['dog','cat','frog','antelope']

# the following line is going to spit out an index out of range error
print(myList[4])